frontend/src/concepts/roleBinding/RoleBindingPermissionsTableRowAdd.tsx

```diff
@@ -2,35 +2,41 @@ import * as React from 'react';
 import { Tbody, Td, Tr } from '@patternfly/react-table';
 import { Button, Split, SplitItem } from '@patternfly/react-core';
 import { CheckIcon, TimesIcon } from '@patternfly/react-icons';
-import { ProjectSharingRBType, ProjectSharingRoleType } from './types';
-import ProjectSharingNameInput from './ProjectSharingNameInput';
-import ProjectSharingPermissionSelection from './ProjectSharingPermissionSelection';
+import { RoleBindingPermissionsRBType, RoleBindingPermissionsRoleType } from './types';
+import RoleBindingPermissionsNameInput from './RoleBindingPermissionsNameInput';
+import RoleBindingPermissionsPermissionSelection from './RoleBindingPermissionsPermissionSelection';
 
-type ProjectSharingTableRowPropsAdd = {
+type RoleBindingPermissionsTableRowPropsAdd = {
   typeAhead?: string[];
-  type: ProjectSharingRBType;
-  onChange: (name: string, roleType: ProjectSharingRoleType) => void;
+  type: RoleBindingPermissionsRBType;
+  permissionOptions?: {
+    type: RoleBindingPermissionsRoleType;
+    description: string;
+  }[];
+  onChange: (name: string, roleType: RoleBindingPermissionsRoleType) => void;
   onCancel: () => void;
 };
 
-/** @deprecated - this should use ProjectSharingTableRow */
-const ProjectSharingTableRowAdd: React.FC<ProjectSharingTableRowPropsAdd> = ({
+/** @deprecated - this should use RoleBindingPermissionsTableRow */
```

Comment:
Would it be a lot of effort to actually deduplicate these and address this deprecation?

Reply:
i haven't looked into it too much, but would prefer that be another story/concern

Reply:
Sounds good. I'll make a note to open a tech debt followup task.

Reply:
Opened a task to follow up on this: https://issues.redhat.com/browse/RHOAIENG-9457